backend/src/main/java/com/park/utmstack/web/rest/UtmConfigurationParameterResource.java

```diff
@@ -67,7 +68,7 @@ public ResponseEntity<Void> updateConfigurationParameters(@Valid @RequestBody Li
         try {
             Assert.notEmpty(parameters, "There isn't any parameter to update");
             for (UtmConfigurationParameter parameter : parameters) {
-                if(parameter.getConfParamDatatype().equals("email_list") && parameter.getConfParamRegexp() != null){
+                if(parameter.getConfParamDatatype().equals("email_list") && StringUtils.hasText(parameter.getConfParamRegexp())){
```

Comment:
all the configurations values must be checked with the regex in case the regex field is not empty